projects/plugins/vaultpress/composer.json

```diff
@@ -0,0 +1,56 @@
+{
+	"name": "automattic/vaultpress",
+	"description": "VaultPress is a subscription service offering real-time backup, automated security scanning, and support from WordPress experts.",
+	"homepage": "https://vaultpress.com/",
+	"type": "wordpress-plugin",
+	"license": "GPL-2.0-or-later",
+	"support": {
+		"issues": "https://github.com/Automattic/jetpack/issues"
+	},
+	"require": {
+		"automattic/jetpack-logo": "1.5.x-dev",
+		"automattic/jetpack-autoloader": "2.10.x-dev"
+	},
+	"require-dev": {
+		"yoast/phpunit-polyfills": "0.2.0",
+		"automattic/jetpack-changelogger": "^1.2"
+	},
+	"autoload": {
+		"classmap": [
+			"src/"
+		]
+	},
+	"scripts": {
+		"phpunit": [
+			"@composer install",
+			"./vendor/phpunit/phpunit/phpunit --colors=always"
+		],
+		"test-coverage": [
+			"@composer install",
+			"phpdbg -d memory_limit=2048M -d max_execution_time=900 -qrr ./vendor/bin/phpunit --coverage-clover \"$COVERAGE_DIR/clover.xml\""
+		],
+		"test-php": [
+			"@composer phpunit"
+		],
+		"build-production": "echo 'Add your build step to composer.json, please!'",
+		"build-development": "echo 'Add your build step to composer.json, please!'"
```

Comment:
You'll want to do something with these. If nothing else, to do `pnpm run build` because you have some build steps defined in package.json.

Reply:
BTW, that's why the Build job is failing. It needs the stuff that `pnpm run build-production` would do in order to have `vendor/` set up right for the mirroring.

Reply:
Good point. Done.